docker/utv/nginx/nginx.conf

```diff
@@ -1,57 +1,49 @@
 worker_processes 1;
 
-events { worker_connections 1024; }
+events {worker_connections 1024;}
 
 http {
 
-    sendfile on;
+  sendfile on;
 
-    upstream frontendAPI {
-        server backing:8080;
+  server {
+    listen 80;
+    server_name	w3utv-mu01.uio.no;
+    if ($request_uri ! = "/check.txt") {
+      rewrite ^(.*)$ 	https://musit-utv.uio.no$1 permanent;
     }
+  }
 
-    upstream webpackServer {
-        server webpack:8000;
+  server {
+
+    listen 80;
+    server_name	musit-utv.uio.no;
+    if ($http_x_is_secure ! = "True") {
+      rewrite ^(.*)$ https://$server_name$1 permanent;
+    }
+
+    include /etc/nginx/mime.types;
+
+    location / {
+      proxy_pass http://webpack:8000;
+    }
+
+    location /musit {
+      proxy_pass http://backing:8080;
     }
 
-    server {
-        listen 80;
-        server_name	w3utv-mu01.uio.no;
-        if ($request_uri != "/check.txt") {
-            rewrite ^(.*)$ 	https://musit-utv.uio.no$1 permanent;
-        }
+    location /api/auth/ {
+      proxy_pass http://auth:7070/;
```

Comment:
Should only expose the `rest` endpoints until we have secured the `web` endpoints.